contracts/nft/erc721m/clones/ERC721MagicDropMetadataCloneable.sol

```diff
@@ -98,6 +98,13 @@ contract ERC721MagicDropMetadataCloneable is
         return _royaltyBps;
     }
 
+    /// @notice Returns the total number of tokens minted by a specific address.
+    /// @param owner The address to query.
+    /// @return The total number of tokens minted by the specified address.
+    function totalMintedByUser(address owner) public view returns (uint256) {
```

Comment:
Why `owner` not `user`?

Reply:
+1 to user, as ByUser is already in the function name